src/widget/form/groupinviteform.cpp

```diff
@@ -149,8 +153,29 @@ void GroupInviteForm::onGroupInviteRejected()
     int index = inviteLayout->indexOf(groupWidget);
     groupInvites.removeAt(index);
 
-    groupWidget->deleteLater();
-    inviteLayout->removeWidget(groupWidget);
+    deleteInviteButtons(groupWidget);
+}
+
+void GroupInviteForm::deleteInviteButtons(QWidget *widget)
+{
+    QList<QPushButton*> buttons = widget->findChildren<QPushButton*>();
+
+    if (acceptButtons.contains(buttons.at(0)))
```

Comment:
This seems wrong to me, because you always delete the first or second buttons, what if the user has 5 friend requests?
Maybe take a look at addfriendfrom.cpp and see how it's handled there.